production/scrapers/nevada.R

```diff
@@ -62,17 +62,25 @@ nevada_pull <- function(x){
     Sys.sleep(10)
     remDr$findElement(
         "xpath", 
-        str_c("//div[@class='slicerItemContainer']",
-              "/span[@title='Correctional']"))$clickElement()
+        str_c("/html/body/div[6]/div[1]/div/div[2]/div/div[1]/div/div/div[5]/div/div",
+              ""))$clickElement()
     Sys.sleep(10)
     remDr$findElement(
-        "xpath", "//div[@aria-label='Facility Name,  All']")$clickElement()
+        "xpath", 
+        str_c(
+            "//div[@aria-label='Facility Name Slicer Drop down box to ",
+            "select one or more facilities.']"))$clickElement()
     Sys.sleep(10)
+    # old selector
+    #remDr$findElement(
+    #    "xpath", "//div[@aria-label='Facility Name,  All']")$clickElement()
+    #Sys.sleep(10)
```

Comment:
Let's delete the old code -- Code is cheap, our git history will capture what was once there, and it's easier to read without noise.